splice/web/api/adgroup.py

```diff
@@ -0,0 +1,130 @@
+from flask import Blueprint
+from flask_restful import Api, Resource, marshal, fields, reqparse
+
```

Comment:
I think one line is better. But it's subjective and up to you. flake8 isn't complaining at the very least ;-)